pkg/vetter/util/util_test.go

```diff
@@ -122,3 +122,87 @@ var _ = Describe("Converting configmap to SidecarInjectionSpec", func() {
 	})
 
 })
+
+var _ = Describe("Test ProxyStatusPort", func() {
+	It("Finds an override that is not 15020", func() {
+
+		// Actual list from a kubectl get pods ratings-v1-76f4c9765f-n8w45 -o yaml call (so what we should have
```

Comment:
Remove these `kubectl get pods ratings-v1-76f4c9765f-n8w45 -o yaml` as the code is open sourced.

Reply:
Done.